src/components/toolbar/toolbar.js

```diff
@@ -72,6 +72,10 @@ function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
       element.addClass('_md');     // private md component indicator for styling
       $mdTheming(element);
 
+      $mdUtil.nextTick(function () {
```

Comment:
I don't think this should trigger a digest, try passing in false as the second argument.